app/controllers/refinery/search/search_controller.rb

```diff
@@ -0,0 +1,22 @@
+require 'will_paginate/array'
+
+module Refinery
+  module Search
+    class SearchController < ::ApplicationController
+
+      before_filter :find_page
+
+      def show
+        @results = Refinery::Search::SearchEngine.search(params[:query])
+        @results = @results.paginate(page: params[:page], per_page: Refinery::Search.results_per_page)
+      end
```

Comment:
Ok, i didn't know that before_action is the new syntax : https://github.com/rails/rails/commit/9d62e04838f01f5589fa50b0baa480d60c815e2c. Thanks :)